### DISCUSSION TITLE
Flashing Sonoff Smart Switch 4Channel Pro R3

### DISCUSSION THREAD

**Discussion by muckisg** [arendst/Tasmota#10852] [Show and tell]
Just as information for the wiki, with revision 3 it is very easy to enter flash mode by pressing S1 when powering io pins.
**Board Info:**
Sonoff 4CH Pro R3
V1.0
2020-04-13

Hope it helps,

Mucki

**Comment by meingraham** [comment]
Please add this note to the docs. You can click the "pencil mark" to edit the article and make a PR.

**Reply by muckisg** [reply]
OK, gonna try it ... :-)
